dataeng/resources/tableau-config-backup.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+set -ex
+
+ssh -tt $USER_NAME@$TABLEAU_SERVER_IP -o StrictHostKeyChecking=no "S3_BUCKET=$S3_BUCKET" '
+
+# It gets the current date from system in 2022.04.05-10:35 format
+DATE="$(date '+%Y.%m.%d')"
+
+# It adds the date as prefix on backup name. Backup runs at 09:00 UTC so it also prefixed in the name.
+BACKUPNAME="${DATE}-tableau-config-backup.json"
+
+sudo runuser -l tsm_admin -c "tsm settings export -f /home/tableau_backups/tableau-config-backup.json"
```

Comment:
shouldn't you be exporting to $BACKUPNAME?

Reply:
Yes, I tried doing it. Since this command has to run as `tsm_admin` (user which is used to run tsm commands). So exporting an environment variable in user  and accessing into another user required additional steps. That's why I thought to simply write a name of backup and append date while I am uploading it on s3. There is only one local copy of the backup so doesn't need to be appended with `date/time`